test/studies/lcals/LongDouble.chpl

```diff
@@ -0,0 +1,39 @@
+/* A module to provide a "long double" type to Chapel that matches the
```

Comment:
I agree with your sentiment about making this a first class module.  I started down the path of implementing a bunch of operator overloads to make it look like other real types, but then backed off and made it more minimal for now.  I made sure to keep it in its own module so that additional support could be easily added and then moved to modules/internal or modules/standard later.